DevEdu.API/DevEdu.Business.Tests/CourseServiceTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿using DevEdu.Business.Exceptions;
+using DevEdu.Business.Services;
+using DevEdu.DAL.Models;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace DevEdu.Business.Tests
+{
+    public class CourseServiceTests
+    {
+        private  Mock<ICourseRepository> _courseRepositoryMock;
+        private  Mock<ITopicRepository> _topicRepositoryMock;
+        private  Mock<ITaskRepository> _taskRepositoryMock;
+        private  Mock<IMaterialRepository> _materialRepositoryMock;
+        private const string _validationExceptionMessageSamePositions = "the same positions of topics in the course";
+        private const string _validationExceptionMessageSameTopics = "the same topics in the course";
```

Comment:
close, but not enough)
the idea is to have only one constant per message. and you this constant in both test and service methods